src/components/student-data-form/student-data-form.js

```diff
@@ -381,12 +386,12 @@ class StudentDataForm extends React.Component {
               key="googleDocsUrl"
               type="text"
               label="Google Docs URL"
-              placeholder="Enter student's google documents url"
+              placeholder="Enter student&rsquo;s Google documents url"
               value={this.state.googleDocsUrl ? this.state.googleDocsUrl : ''}
               onChange={this.handleTextFieldChange}
             />
           </FormGroup>
-          <Button type="submit" className="formSubmitBtn" id="submit-student-data">Submit</Button>
+          <Button type="submit" className="formSubmitBtn" id="submit-student-data" onClick={this.props.updateStudentData}>Submit</Button>
```

Comment:
Okay. It seems like nothing is happening upon submit, because the modal stays open and there was no alert. It would be great from a UI perspective to have something happen.